boards/adi/max32657evkit/max32657evkit_max32657.yaml

```diff
@@ -10,5 +10,6 @@ toolchain:
 supported:
   - serial
   - trng
+  - gpio
```

Comment:
why does this need to be it's own commit to edit 2 yaml files?

Reply:
Just to give credit to commiter. Ok this simple update merged.